src/lib/spinel/spinel.h

```diff
@@ -4959,6 +4959,18 @@ enum
      */
     SPINEL_PROP_INFRA_IF_SETUP = SPINEL_PROP_INFRA_IF__BEGIN + 1,
 
+    /// Infrastructure interface state update.
+    /** Format: `bA(6)`
+     * Type: Write
+     *
+     * `b`: If the infrastrue interface is running.
+     * `A(6)`: The IPv6 addresses of the infrastructure interface.
+     *
+     * This property is used when the host infrastructure interface has an update of the UP/DOWN state or IP addresses.
+     *
+     */
+    SPINEL_PROP_INFRA_IF_STATE = SPINEL_PROP_INFRA_IF__BEGIN + 2,
```

Comment:
is `SPINEL_PROP_INFRA_IF_SETUP` sufficient?

Reply:
Thanks! As per discussion offline, have reused the existing spinel property.

However I changed the name to `INFRA_IF_STATE` because this is a spinel property and STATE is more like a property. Thoughts?

Reply:
SPINEL_PROP_INFRA_IF_SETUP looks good to me, I don't have strong opinion if you want to rename.